.github/workflows/publish_master.yml

```diff
@@ -38,4 +38,4 @@ jobs:
       run: |
         cd $GITHUB_WORKSPACE/src/DotNetLightning.Core
         dotnet pack . -p:Configuration=Release --version-suffix date`date +%Y%m%d-%H%M`-git-`echo $GITHUB_SHA | cut -c 1-7`-${{ matrix.RID }}
-        dotnet nuget push ./bin/Release/DotNetLightning.Core.1*.nupkg -k ${{ secrets.NUGET_API_KEY }} -s https://api.nuget.org/v3/index.json
\ No newline at end of file
+        dotnet nuget push ./bin/Release/DotNetLightning.Kiss.1*.nupkg -k ${{ secrets.NUGET_API_KEY }} -s https://api.nuget.org/v3/index.json
```

Comment:
this looks wrong, we should push BouncyCastle binaries, not native

Reply:
Oops. I've fixed it so that we only upload the BouncyCastle build and we upload it as DotNetLightning.Kiss